torch/_prims/context.py

```diff
@@ -301,12 +313,15 @@ def __torch_function__(
         if self._is_var_mean(orig_func):
             return torch.ops.nvprims.var_mean(*args, **kwargs)
 
+        if self._is_view_or_reshape(orig_func):
+            return torch.ops.nvprims.view(*args, **kwargs)
```

Comment:
We need this to support `tensor.view(1, 2, 3, 4, 5, 6)`
```suggestion
        if self._is_view_or_reshape(orig_func):
            a, *shape = args
            shape = torch._prims_common.extract_shape_from_varargs(shape, validate=False)
            if len(kwargs) > 0:
                warn("view has ignored kwargs!")
            return torch.ops.nvprims.view(a, shape)
```